lua/package-info/config.lua

```diff
@@ -95,14 +95,14 @@ end
 --- Register autocommand for loading the plugin
 -- @return nil
 M.__register_start = function()
-    register_autocmd("BufEnter", "lua require('package-info.core').load_plugin()")
+    register_autocmd("BufEnter", "package.json", "lua require('package-info.core').load_plugin()")
```

Comment:
I'd actually like the load the plugin regardless of which file is it since I like to quickly install packages when im working on other files and I've never visited `package.json` file.

Reply:
Is what I have now good? Or does the other one also need to be a wildcard? What I have now still doesn't require the message that gets displayed to be removed. If the other needs to be a wildcard, I'll remove the logging message again.